react/src/components/MainLayout/MainLayout.tsx

```diff
@@ -217,6 +218,13 @@ function MainLayout() {
                 <ForceTOTPChecker />
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +181 to +207]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Please delete this comment :) <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2847?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->